examples/mwaa/environment.yaml

```
@@ -0,0 +1,256 @@
+apiVersion: mwaa.aws.upbound.io/v1beta1
+kind: Environment
+metadata:
+  annotations:
+    meta.upbound.io/example-id: mwaa/v1beta1/environment
+  labels:
+    testing.upbound.io/example-name: example
+  name: example-environment
+spec:
+  forProvider:
+    dagS3Path: dags/
+    executionRoleArnSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example_role
+    networkConfiguration:
+      securityGroupIdsRefs:
+        - name: example-security-group
```

It would be great to use a selector.

```suggestion
      securityGroupIdsSelector:
        matchLabels:
          testing.upbound.io/example-name: example_security_group
```